docs/reference/2-ods-api/platform-dev-guide/features/ownership-based-authorization.md

```diff
@@ -273,4 +274,25 @@ used to assign an ownership token to each record.
 
 Eg. Student Table
 
-![Ed-Fi ODS Database](../../../../../static/img/reference/ods-api/image2022-2-3_10-37-3.png)
+```mermaid
+
+%%{init: {
+    "theme": "",
+    "themeCSS": [
+        "[id*=entity-Student] rect:nth-of-type(24) { fill: orange;}",
+        "[id*=entity-Student] rect:nth-of-type(11), rect:nth-of-type(12), rect:nth-of-type(13) { fill: silver;}"
+    ]
+}
+}%%
+erDiagram
+    Student {
+        int StudentUsi PK
+        varchar StudentUniqueId
+        varchar FirstName
+        etc etc
+        datetime CreateDate
+        datetime LastModifiedDate
+        uniqueidentifier Id
+        int CreatedByOwnershipTokenId
+    }
```

Comment:
Here's an example of the highlighting:

![image](https://github.com/user-attachments/assets/a76a00dd-f064-4b10-b0b7-5c0aa9978dfd)

The silver color for "etc" was supposed to be like "graying out the text". I don't know how well that worked. Text color cannot be changed with mermaid.